src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -164,8 +164,9 @@ public function getInfo(string $type = null)
 
             rewind($this->debugBuffer);
             $info['debug'] = stream_get_contents($this->debugBuffer);
+            $waitFor = curl_getinfo($this->handle, CURLINFO_PRIVATE);
 
-            if (!\in_array(curl_getinfo($this->handle, CURLINFO_PRIVATE), ['headers', 'content'], true)) {
+            if ('H' !== $waitFor[0] && 'B' !== $waitFor[0]) {
```

Comment:
Hmm should it be a C instead of  B ?

Reply:
I'm using B for body, could be C for content. It depends on the moon on my side :P